src/seedu/addressbook/data/person/Phone.java

```diff
@@ -6,27 +6,25 @@
  * Represents a Person's phone number in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
  */
-public class Phone {
+public class Phone extends Contact {
 
     public static final String EXAMPLE = "123456789";
     public static final String MESSAGE_PHONE_CONSTRAINTS = "Person phone numbers should only contain numbers";
     public static final String PHONE_VALIDATION_REGEX = "\\d+";
 
-    public final String value;
-    private boolean isPrivate;
+    //public final String value;
+    //private boolean isPrivate;
```

Comment:
Don't leave code commented out. The repo already helps you retrieve deleted code.